src/main/kotlin/org/rust/cargo/toolchain/tools/Rustfmt.kt

```diff
@@ -147,7 +148,8 @@ class Rustfmt(toolchain: RsToolchainBase) : RustupComponent(NAME, toolchain) {
         private fun RsProcessExecutionException.showRustfmtError(project: Project) {
             val message = message.orEmpty().trimEnd('\n')
             if (message.isNotEmpty()) {
-                project.showBalloon("Rustfmt", message, NotificationType.ERROR, RustfmtEditSettingsAction("Show settings..."))
+                val html = "<html>${escapeHtml(message).replace("\n", "<br>")}</html>"
```

Comment:
Let's better use `org.rust.openapiext.escape` instead of `org.apache.commons.lang.StringEscapeUtils.escapeHtml` to avoid additional dependency to 3rd party lib